src/Http/Requests/AttachCategoriesCourseAPIRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace EscolaLms\Courses\Http\Requests;
+
+use EscolaLms\Courses\Models\Course;
+use Illuminate\Foundation\Http\FormRequest;
+
+class AttachCategoriesCourseAPIRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
If you don't check anything in authorize, you can just remove this function as it returns true by default in FormRequest